drivers/sensor/grow_r502a/grow_r502a.c

```diff
@@ -287,6 +288,7 @@ static int fps_image_to_char(const struct device *dev, uint8_t char_buf_idx)
 	struct grow_r502a_data *drv_data = dev->data;
 	union r502a_packet rx_packet = {0};
 	char const img_to_char_len = 2;
+	int ret = 0;
```

Comment:
Are the `ret` changes in this commit related to `drivers: sensor: grow_r502a: move enroll, search, delete to application`? It looks like they may belong in a separate commit